### ISSUE TITLE
[Tarea 2] Pregunta 2 - Función inverso

### ISSUE THREAD

**Issue by VicenteMerino** [UC-IIC3253/2021#34]
Profesor, para la implementación de la función `inverso`, debemos retornar siempre un inverso positivo? O es válido que sea positivo o negativo?

**Comment by marceloarenassaavedra** [comment]
Sí, se debe retornar el inverso positivo que está entre `0` y `n-1`. El enunciado de la tarea ahora indica que `n >= 2`, dado que queremos calcular inversos para números que satisfacen esta restricción.

Saludos!

Marcelo

**Comment by lfuenzalida2** [comment]
Podemos asumir que "a" siempre va a ser menor que "n"?? Lo digo porque en el algoritmo de euclides se asume que los números tienen un orden y en esta función uso el algoritmo de euclides
